linalg/densemat.hpp

```diff
@@ -58,6 +58,20 @@ class DenseMatrix : public Matrix
    DenseMatrix(double *d, int h, int w)
       : Matrix(h, w) { UseExternalData(d, h, w); }
 
+   /// Create a dense matrix using a braced initializer list
```

Comment:
Maybe the Doxygen comment should mention that each _inner_ list represents a row in the matrix.